.github/workflows/make.yml

```diff
@@ -30,4 +31,10 @@ jobs:
         sudo make PREFIX=/usr install
     - name: Make release
       run: |
-        make release
+        # TODO: Figure out release without requiring root
+        sudo make release
```

Comment:
Looking at the action's log, it is odd that creating a virtualenv results in a permission error, I don't understand why that happens either.